index.js

```diff
@@ -21,7 +21,7 @@ var MAX_PARALLEL_TASKS = 5;
  *      ownership of a key, for example if another node now owns it.  Cleanup.
  *      Also takes a 'done' callback.
  */
-function Sevnup(loadVNKeysFromStorage,
+function Sevnup(loadVNKeysFromStorage, // jshint ignore:line
```

Comment:
gives me a too many arguments complaint.  the alternative is to just make an amorphous object with an equal amount of arguments that you just have to pack and unpack, so it's moot.